currency_rate_update/models/res_currency_rate_provider_ECB.py

```diff
@@ -3,62 +3,95 @@
 # Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
+import xml.sax
 from collections import defaultdict
 from datetime import date, timedelta
 from urllib.request import urlopen
-import xml.sax
 
-from odoo import models, fields, api, _
+from odoo import _, api, fields, models
 from odoo.exceptions import UserError
 
 
 class ResCurrencyRateProviderECB(models.Model):
-    _inherit = 'res.currency.rate.provider'
+    _inherit = "res.currency.rate.provider"
 
-    service = fields.Selection(
-        selection_add=[('ECB', 'European Central Bank')],
-    )
+    service = fields.Selection(selection_add=[("ECB", "European Central Bank")])
 
-    @api.multi
+    @api.model
```

Comment:
Why?

Reply:
It supposed to be run on entity, not on class

Reply:
@alexey-pelykh Right, reverted